source/renderer/util/load-i18n.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
First, I would suggest to keep this file in the language-folder, to keep all language-related things together. Besides, this file is not only used by the renderer process (that is: the main window), but by others as well.  Therefore, the `common/lang`-directory is better for this function.

Then, I'm not sure whether or not `load-i18n` is a good name, because `i18n()` and `loadI18n()` in principle do exactly the same thing, only for different process types. Maybe keep `i18n()`, or rename it to `i18nMain()`, and name the other one `i18nRenderer()`, indicating the different processes this is being used in.

Last but not least: If we keep it in a separate file (which I would suggest), please copy over a file header from one of the other files, adapting it w/ your name/user-name/other nickname, optionally a mail, to indicate this was your work and to make sure we have the GNU GPL v3-license in the file!

Besides that, I'm pretty happy with it, and as it works for you, I'd happily merge it!

Reply:
Here it is!
I've fixed all yours points, although I think we could have kept the new function into i18n.js as well.

Small question for the next time: should I update the CHANGELOG.md for each pull request , or you do it once per version update ?

Reply:
Please feel free to also update the Changelog!

In general, I tend to put visible improvements in user experience as "new features", everything else that users might notice into "GUI and Functionality" and everything users might not be able to see into "Under the Hood", so this issue would go to the latter section!

Plus, as you're a contributor, please feel free to add your GitHub handle (I've done this in the 1.5 Changelog, just have a look how I mentioned contributors there) :)

Reply:
Also, when changes are required, I always rebase my commits out of habit, instead of adding more commits to fix the previous ones. Would you prefer I keep the history ?

Reply:
Eeeeeeeh … I don't know — I mean having more history means that in the worst case we can track down issues, right? I've never done a rebase, I tend to only glue all commits to the HEAD of a branch without merging them together, so … I don't know. For this particular instance, it should not be a problem, but in general I would say let's try to keep as much history as possible!